ChangeLog

```diff
@@ -1,3 +1,155 @@
+ - config: drop refresh_rmc_and_interface as RHEL 7 no longer supported
+   [Robert Schweikert]
+ - docs: Add feedback button to docs
+ - net/sysconfig: enable sysconfig renderer if network manager has ifcfg-rh
+   plugin (#4132) [Ani Sinha]
+ - For Alpine use os-release PRETTY_NAME (#4138) [dermotbradley]
+ - network_manager: add a method for ipv6 static IP configuration (#4127)
+   [Ani Sinha]
+ - correct misnamed template file host.mariner.tmpl (#4124) [dermotbradley]
+ - nm: generate ipv6 stateful dhcp config at par with sysconfig (#4115)
+   [Ani Sinha]
+ - Add templates for GitHub Issues
+ - Add 'peers' and 'allow' directives in cc_ntp (#3124) [Jacob Salmela]
+ - FreeBSD: Fix user account locking (#4114) [Mina Galić] (GH: #1854594)
+ - FreeBSD: add ResizeGrowFS class to cc_growpart (#2334) [Mina Galić]
+ - Update tests in Azure TestCanDevBeReformatted class (#2771)
+   [Ksenija Stanojevic]
+ - Replace Launchpad references with GitHub Issues
+ - Fix KeyError in iproute pformat (#3287) [Dmitry Zykov]
+ - schema: read_cfg_paths call init.fetch to lookup /v/l/c/instance
+ - azure/errors: introduce reportable errors for imds (#3647)
+   [Chris Patterson]
+ - FreeBSD (and friends): better identify MBR slices (#2168)
+   [Mina Galić] (LP: #2016350)
+ - azure/errors: add host reporting for dhcp errors (#2167)
+   [Chris Patterson]
+ - net: purge blacklist_drivers across net and azure (#2160)
+   [Chris Patterson]
+ - net: refactor hyper-v VF filtering and apply to get_interfaces() (#2153)
+   [Chris Patterson]
+ - tests: avoid leaks to underlying filesystem for /etc/cloud/clean.d
+   (#2251)
+ - net: refactor find_candidate_nics_on_linux() to use get_interfaces()
+   (#2159) [Chris Patterson]
+ - resolv_conf: Allow > 3 nameservers (#2152) [Major Hayden]
+ - Remove mount NTFS error message (#2134) [Ksenija Stanojevic]
+ - integration tests: fix image specification parsing (#2166)
+ - ci: add hypothesis scheduled GH check (#2149)
+ - Move supported distros list to docs (#2162)
+ - Fix logger, use instance rather than module function (#2163)
+ - README: Point to Github Actions build status (#2158)
+ - Revert "fix linux-specific code on bsd (#2143)" (#2161)
+ - Do not generate dsa and ed25519 key types when crypto FIPS mode is
+   enabled (#2142) [Ani Sinha] (LP: 2017761)
+ - Add documentation label automatically (#2156)
+ - sources/azure: report success to host and introduce kvp module (#2141)
+   [Chris Patterson]
+ - setup.py: use pkg-config for udev/rules path (#2137) [dankm]
+ - openstack/static: honor the DNS servers associated with a network
+   (#2138) [Gonéri Le Bouder]
+ - fix linux-specific code on bsd (#2143)
+ - cli: schema validation of jinja template user-data (SC-1385) (#2132)
+   (LP: #1881925)
+ - gce: activate network discovery on every boot (#2128)
+ - tests: update integration test to assert 640 across reboots (#2145)
+ - Make user/vendor data sensitive and remove log permissions (#2144)
+   (LP: #2013967)
```

Comment:
@TheRealFalcon This content is officially released already as 23.1.2. shall we redact it

Reply:
Opted to leave duplicated content in "security releases" given that most reviewers of 23.2 will be looking at wholesale set of changes between 23.2 and 23.1... and having the hot-fixes included in 23.2 section makes sense.